.github/workflows/stockfish.yml

```diff
@@ -10,6 +10,8 @@ on:
       - master
       - tools
 jobs:
+  Binaries:
+    uses: Disservin/Stockfish/.github/workflows/stockfish_binaries.yml@artifact-sha-sep
```

Comment:
this points to Disservin repo?

Reply:
Yes since I don’t know if it can point to pull request and as of now this file doesn’t exist in stockfish master

Reply:
This would be something you need to change when a merge is happening

Reply:
Why don't you simply run the workflow on push to master?

Reply:
So that you can see how it would look like ?^^ However, now that you have seen it I can change it so that the binaries are only created on push and change the link to official stockfish